File: app/src/main/java/com/example/newdoctorsapp/models/CreateDoctor/Registration.java
package com.example.newdoctorsapp.models.CreateDoctor;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Registration {

    @SerializedName("registrationNumber")
    @Expose
    private String registrationNumber;
    @SerializedName("registrationCouncil")
    @Expose
    private String registrationCouncil;
    @SerializedName("registrationDate")
    @Expose
    private String registrationDate;

    public String getRegistrationNumber() {
        return registrationNumber;
    }

    public void setRegistrationNumber(String registrationNumber) {
        this.registrationNumber = registrationNumber;
    }

    public String getRegistrationCouncil() {
        return registrationCouncil;
    }

    public void setRegistrationCouncil(String registrationCouncil) {
        this.registrationCouncil = registrationCouncil;
    }

    public String getRegistrationDate() {
        return registrationDate;
    }

    public void setRegistrationDate(String registrationDate) {
        this.registrationDate = registrationDate;
    }

}

File: app/src/main/java/com/example/newdoctorsapp/models/ProfileDetail/Registration.kt
package com.example.newdoctorsapp.models.ProfileDetail

import com.google.gson.annotations.SerializedName

data class Registration(
        @SerializedName("_id") val _id: String,
        @SerializedName("registrationCouncil")  val registrationCouncil: String,
        @SerializedName("registrationDate") val registrationDate: String,
        @SerializedName("registrationNumber") val registrationNumber: String
)